manage.py

```diff
@@ -588,11 +601,14 @@ def stale():
     skip = os.getenv('KINGFISHER_SUMMARIZE_PROTECT_SCHEMA', '').split(',')
 
     statement = """
-        SELECT 1 FROM {schema}.selected_collections sc JOIN collection c ON sc.id = c.id WHERE deleted_at IS NULL
+        SELECT DISTINCT schema
+        FROM summaries.selected_collections sc
+        JOIN collection c ON sc.collection_id = c.id
+        WHERE deleted_at IS NOT NULL
     """
 
-    for schema in db.schemas():
-        if schema not in skip and not db.one(statement, schema=schema):
+    for schema in db.pluck(statement):
+        if schema not in skip:
```

Comment:
This logic is not the same. The old code *skips* a summary as long as there is at least one *non-deleted* collection. The new code *prints* a summary as long as there is at least one *deleted* collection.

We want to only print a summary if all its collections are deleted.

I think the way to do that is to change the query to `IS NULL`, to get the results of the query outside the loop, to loop over all schemas, and then do `and schema not in results`.